components/NavBar/navbar.module.css

```diff
@@ -65,6 +65,10 @@
 .userGreet {
   color: #ffffff;
   margin: 23px;
+  cursor: pointer;
+}
+.userGreet:hover {
+  color: #49a82e;
```

Comment:
You can use variables for this color as well

Reply:
Sure I will use 👍🏻